src/api/index.js

```diff
@@ -0,0 +1 @@
+export const workFlowsCountUrl = "https://bc-cms-rewrite-node-ms-git-apis-raj-kumar-litmus-s-team.vercel.app/api/v1/workflows/counts"
```

Comment:
here the domain name `https://bc-cms-rewrite-node-ms-git-apis-raj-kumar-litmus-s-team.vercel.app` should not be hardcoded.
Keep it in the .env file and load it here.

Reply:
I thought this file is going to have all the fetch logic.
If you just need to keep the urls.
create a folder called constants/index.js and add the urls there.

Reply:
Done